app/views/articles/_sidebar_nav.html.erb

```diff
@@ -8,53 +8,105 @@
+      Podcasts
+    </a>
+    
+    <a href="#" class="crayons-link crayons-link--secondary crayons-link--block crayons-link--block--indented fs-s" id="main-nav-more-trigger">More...</a>
+    
+    <div class="hidden" id="main-nav-more">
+      <a href="/code-of-conduct" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/thumb-up.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Code of Conduct") %>
+        Code of Conduct
+      </a>
+      <a href="/faq" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/bulb.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "FAQ") %>
+        FAQ
+      </a>
+      <a href="https://shop.dev.to/" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/shopping.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Shop") %>
+        <%= ApplicationConfig["COMMUNITY_NAME"] %> Shop
+      </a>
+      <a href="/sponsors" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/heart.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Sponsors") %>
+        Sponsors
+      </a>
+      <a href="/about" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("rainbowdev.svg", aria: true, width: 24, height: 24, class: "crayons-icon crayons-icon--default p-1", title: "About") %>
+        About
+      </a>
+      <a href="/privacy" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/smart.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "FAQ") %>
+        Privacy policy
+      </a>
+      <a href="/terms" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/look.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "FAQ") %>
+        Terms of use
+      </a>
+      <a href="/contact" class="crayons-link crayons-link--block">
+        <%= inline_svg_tag("twemoji/contact.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "FAQ") %>
+        Contact
+      </a>
+      <div class="grid gap-2 grid-cols-5 p-4 mt-4 border-solid border-0 border-t-1 border-base-10">
+        <% SiteConfig.social_media_handles.keys.each do |social_media_type| %>
+          <% if SiteConfig.social_media_handles[social_media_type].present? %>
+            <a href="<%= "https://#{social_media_type}.com/#{SiteConfig.social_media_handles[social_media_type]}" %>" target="_blank" rel="noopener" class="crayons-link crayons-link--secondary">
+              <%= inline_svg_tag("#{social_media_type}.svg", aria: true, title: "#{social_media_type.capitalize}", class: "crayons-icon") %>
             </a>
-          </div>
+          <% end %>
         <% end %>
       </div>
     </div>
-    <a href="<%= tags_path %>" class="cta cta-button">View all-time top tags</a>
-  </div>
+  </nav>
+
+  <% if user_signed_in? %>
```

Comment:
We should keep list of tags on home page in some capacity for SEO purposes. Help bots crawl to the tag pages more effectively.

Reply:
@benhalpern I’m not really touching tags section right now except moving “View all tags” link to the main nav